.github/workflows/ci.yml

```diff
@@ -85,6 +85,52 @@ jobs:
         chmod +x codecov
         ./codecov
 
+  #
+  # Android build job
```

Comment:
FYI https://github.com/mockito/mockito/pull/2894
(I don't think it's necessary for this pipeline PR, but was inspired :) )